Projects/examples/MODIS.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='utf-8'?>
+<Project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+xsi:noNamespaceSchemaLocation="../XSD/V1/MODIS.xsd">
+<Name>MODIS for HUC 1706</Name>
+<ProjectType>MODIS</ProjectType>
+<MetaData>
+  <Meta name="ModelVersion">0.0.1</Meta>
+  <Meta name="dateCreated">2020-10-28T15:08:11.579051</Meta>
+  <Meta name="HUC4">1706</Meta>
+  <Meta name="HUC">1706</Meta>
+</MetaData>
+<Realizations>
+  <MODIS id="MODIS1" dateCreated="2020-10-27T15:08:11.598597" guid="1cba8df3-f576-432d-b784-6018220224f2" productVersion="0.0.1">
+    <Name>MODIS for HUC 1706</Name>
+    <Outputs>
+      <ZipFile>
+        <Name>MODIS Results</Name>
+        <Path>1706.zip</Path>
```

Comment:
Any metadata?

Reply:
Replace all references to MODIS with LST